components/tikv_alloc/src/error.rs

```diff
@@ -6,7 +6,8 @@ pub enum ProfError {
     MemProfilingNotEnabled,
     IOError(std::io::Error),
     JemallocError(i32),
-    PathError(std::ffi::OsString), // When temp files are in a non-unicode directory, OsString.into_string() will cause this error
+    PathEncodingError(std::ffi::OsString), // When temp files are in a non-unicode directory, OsString.into_string() will cause this error,
+    PtahError(std::ffi::NulError),
```

Comment:
Typo: `Ptah` -> `Path`
And I think the name `PathError` is too vague. As it's caused by a pathname with `\0`, I think it can be `PathWithNulError`.

Reply:
Fixed. Thanks for your review. PTAL~